modules/setup/src/main/Mappings.scala

```diff
@@ -14,7 +15,7 @@ object Mappings {
   val aiVariants = number.verifying(Config.aiVariants contains _)
   val variantWithVariants = number.verifying(Config.variantsWithVariants contains _)
   val variantWithFenAndVariants = number.verifying(Config.variantsWithFenAndVariants contains _)
-  val time = number.verifying(HookConfig validateTime _)
+  val time = of[Double].verifying(HookConfig validateTime _)
```

Comment:
Does this really still work? `validateTime` isn't changed to take `Double` https://github.com/ornicar/lila/blob/b73800ddb46eff91a64963fc9b895cbb893b4fb4/modules/setup/src/main/Config.scala#L117